kolibri/plugins/coach/assets/test/showPageActions.spec.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env mocha */
+const sinon = require('sinon');
+const kolibri = require('kolibri');
+
+// Add fake Resources to kolibri mock. This needs to be done before loading tested module
+kolibri.resources = {
+  ChannelResource: {
+    getCollection: sinon.stub(),
+  },
+  ClassroomResource: {
+    getModel: sinon.stub(),
+  },
+  ContentNodeResource: {},
+  ExamResource: {
+    getCollection: sinon.stub(),
+  },
+  LearnerGroupResource: {
+    getCollection: sinon.stub(),
+  },
+};
+
+const examActions = require('../src/state/actions/exam');
```

Comment:
I think once we start writing more vuex tests involving `kolibri`, I'll make `kolibriGlobalMock` a proper mock so we don't have to do this kind of setup in tests.